lib/components/molecules/Form/FieldFileInput/FieldFileInput.style.js

```diff
@@ -0,0 +1,18 @@
+import { css } from 'styled-components';
+
+// TODO: Fix the duplicate styling in input.style.js
+const styles = css`
+  margin-bottom: 0.75rem;
```

Comment:
We should be using margins and paddings from variables

Reply:
I changed to variabled for colors, spacing wherever applicable.